openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/data_insight.py

```diff
@@ -39,7 +39,8 @@ def build_data_insight_workflow_config(
 
     workflow_config = OpenMetadataWorkflowConfig(
         source=build_source(ingestion_pipeline),
-        sink=ingestion_pipeline.sink,
+        # ingestion_pipeline.service.serviceConnection.elasticsearch
```

Comment:
```suggestion
```

Reply:
will be handled in https://github.com/open-metadata/OpenMetadata/pull/8729